clusters/build-clusters/01_cluster/cert-manager/cert-manager.yaml

```diff
@@ -0,0 +1,175 @@
+apiVersion: apps/v1
```

Comment:
Why do we have a cert-manager.yaml and a _cert-manager.yaml?

Reply:
Those are taken from upstream.
We split into 2 files before the shared part is in common folder (CDs and RBACs etc) and the customization for each cluster (mainly on deployment, like anti-affinity you suggested above) is in its own folder.

Reply:
Renamed the one in cluster folder.